src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockFruitTreeBranch.java

```diff
@@ -39,9 +40,29 @@
     /* Facing of this branch */
     public static final PropertyEnum<EnumFacing> FACING = PropertyEnum.create("facing", EnumFacing.class);
 
-    private static final AxisAlignedBB VERTICAL_AABB = new AxisAlignedBB(0.3125D, 0.0D, 0.3125D, 0.6875D, 1.0D, 0.6875D);
-    private static final AxisAlignedBB HORIZONTAL_Z_AABB = new AxisAlignedBB(0.375D, 0.375D, 0.0D, 0.625, 0.625, 1.0D);
-    private static final AxisAlignedBB HORIZONTAL_X_AABB = new AxisAlignedBB(0.0D, 0.375D, 0.375D, 1.0D, 0.625, 0.625);
+    /* Connection sides
+     * 0 = no connection
+     * 1 = connected, use vertical model
+     * 2 = connected=, use horizontal model */
+    public static final PropertyInteger NORTH = PropertyInteger.create("n", 0, 2);
+    public static final PropertyInteger EAST = PropertyInteger.create("e", 0, 2);
+    public static final PropertyInteger SOUTH = PropertyInteger.create("s", 0, 2);
+    public static final PropertyInteger WEST = PropertyInteger.create("w", 0, 2);
+    public static final PropertyInteger UP = PropertyInteger.create("u", 0, 2);
```

Comment:
These should use actual names, i.e. "north", "south", "up", for consistency with vanilla, and general readability.